src/views/product-downloads-view/helpers.ts

```diff
@@ -277,19 +322,22 @@ export const sortAndFilterReleaseVersions = ({
 			}
 
 			// Filter out prereleases
-			const isPrelease = semverPrerelease(version) !== null
-			if (isPrelease) {
+			const isPrerelease = semverPrerelease(version) !== null
+			if (isPrerelease) {
 				return false
 			}
 
-			// Filter in enterprise versions if enterprise mode
-			const isEnterpriseVersion = !!version.match(/\+ent(?:.*?)*$/)
+			/**
+			 * Filter in enterprise versions if enterprise mode (build === "ent"),
+			 * or filter out any custom "build" values otherwise.
+			 */
 			if (isEnterpriseMode) {
+				const isEnterpriseVersion = getIsEnterpriseVersion(version)
 				return isEnterpriseVersion
+			} else {
+				const { build } = semverParse(version)
+				return build.length === 0
```

Comment:
Whoops, just noticed this as well - the regular downloads page in earlier previews on this PR showed some pre-release-ish enterprise versions, since eg `1.11.0+ent-rc` is identified as having `build: [ "ent-rc" ]` and `prerelease: []`, which means it's _not_ identified as a prerelease _or_ as an enterprise version. This updated logic ensures _any_ version string with a build value (eg `x.y.z+build`) will get filtered out of our dropdown.